docs/api-reference/static-analysis.md

```diff
@@ -24,13 +26,13 @@ DBOS uses several techniques to ensure that static analysis is as productive as
 
 [`eslint`](https://eslint.org/) is a popular tool for checking JavaScript and TypeScript code.  `eslint` is flexible, extensible, and comes with many standard and optional plugins.  Many editors and development tools provide integration with `eslint`, allowing bugs to be detected early in the development cycle.
 
-Many DBOS=suggested coding practices can be enforced by a combination of `eslint` plugins and rule configurations.
+Many DBOS-suggested coding practices can be enforced by a combination of `eslint` plugins and rule configurations.
 
-### Installing And Configuring `eslint`
+### Installing and configuring `eslint`
 
 ::::tip
 If you got started with the [quickstart](../getting-started/quickstart.md), `eslint` and required plugins are already installed.
-Plugins to support TypeScript and detect common vulerabilities are automatically installed with `@dbos-inc/eslint-plugin` as dependencies and do not need to be installed separately.
+Plugins to support TypeScript and detect common vulnerabilities are automatically installed with `@dbos-inc/eslint-plugin` as dependencies and do not need to be installed separately.
 ::::
 
 To install the `eslint` package and the DBOS plugin:
```

Comment:
Update this with ESLint 8 and 9 tabs referencing the flat config (https://docs.dbos.dev/getting-started/quickstart)

Reply:
Done